gatsby-config.js

```diff
@@ -13,29 +13,45 @@ const queries = require("./src/utils/algolia");
 const getPlugins = () => {
   const plugins = [
     {
-      resolve: "@sentry/gatsby",
+      resolve: '@sentry/gatsby',
```

Comment:
did the prettier config (thats present in the repo) do this? i worry that if i open it its going to change it all back

Reply:
I run Prettier automatically but I'll make sure the config from the repo was used for this.

Reply:
Just ran `yarn format`, same result. It even changes 14 more files.

Reply:
hrm maybe we have a fucked up config in develop repo :s

this is what we have in docs

```
{
  "trailingComma": "es5",
  "tabWidth": 2,
  "semi": true,
  "singleQuote": false,
  "overrides": [
    {
      "files": "*.mdx",
      "options": {
        "parser": "markdown"
      }
    }
  ]
}
```

Reply:
is it possible your prettier has some global config? like singleQuotes arent default are they?

i see some of the other formatting issues tho

Reply:
Submitted #151. Will rebase onto that once merged.